deployment/src/test/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/FileConfigSource.java

```diff
@@ -0,0 +1,51 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+final class FileConfigSource implements ConfigSource {
```

Comment:
Are you using this `ConfigSource` to parse the application properties file and put the Circuit Breaker properties in the generator context? Smallrye won't have a similar feature?